lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py

```diff
@@ -414,7 +414,7 @@ def execute_module(self):
         # Changes in VM's spec or any changes to VMIs warrant a full CRUD, the latter because
         # VMIs don't really have states to manage; they're either present or don't exist
         # Also check_mode always warrants a CRUD, as that'll produce a sane result
-        if vm_spec_change or (ephemeral and vm_spec_change) or k8s_state == 'absent' or self.check_mode:
+        if vm_spec_change or k8s_state == 'absent' or self.check_mode:
```

Comment:
Can you please change the comment so it better describe the if

Reply:
Good point. Done.